### PR TITLE
Add `breezy-hr` map for `ListJobs` use case

### PR DESCRIPTION
**Pull request by janhalama**
## Description

This PR adds `breezy-hr` map for `ListJobs` use case.

## Motivation and Context

The PR is based on [feature/add-ats-job-board-demo-use-cases](https://github.com/superfaceai/station/tree/feature/add-ats-job-board-demo-use-cases).

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I haven't repeated the code. (DRY)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by martinalbert** [comment]
@janhalama is base branch `main` intentionally?

**Comment by janhalama** [comment]
> @janhalama is base branch `main` intentionally?

No sorry, I am going to fix it.

File: grid/recruitment/jobs/maps/breezy-hr.suma [hunk 1]
```diff
@@ -0,0 +1,131 @@
+profile = "recruitment/jobs@1.0"
+provider = "breezy-hr"
+
+map ListJobs {
+    http GET "/v3/company/{ parameters.COMPANY_ID }/positions" {
```
Comment (martinalbert) [inline grid/recruitment/jobs/maps/breezy-hr.suma:5]:
```suggestion
  http GET "/v3/company/{ parameters.COMPANY_ID }/positions" {
```

**Comment by janhalama** [comment]
> > @janhalama is base branch `main` intentionally?
> 
> No sorry, I am going to fix it.

@martinalbert  Done. You can review the code.

File: grid/recruitment/jobs/maps/breezy-hr.suma [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            requirements: [],
+            benefits: [],
+            departmentName: job.department,
+          }
+        })
+
+        map result {
+          jobs = jobs
+          rateLimit = call MapRateLimit(headers = headers)
+        }
```
Comment (martinalbert) [inline grid/recruitment/jobs/maps/breezy-hr.suma:30]:
```suggestion
      jobs = body.map(job => {
        return {
          id: job._id,
          name: job.name,
          description: job.description,
          requirements: [],
          benefits: [],
          departmentName: job.department,
        }
      })

      map result {
        jobs = jobs
        rateLimit = call MapRateLimit(headers = headers)
      }
```

File: grid/recruitment/jobs/maps/breezy-hr.suma [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    code = 'RateLimitReached'
+    rateLimit = rateLimit
+  }
+
+  return {
+    title = "Unknown error"
+    detail = workableError
+    code = 'UnknownError'
+    rateLimit = rateLimit
+  }
```
Comment (martinalbert) [inline grid/recruitment/jobs/maps/breezy-hr.suma:130]:
```suggestion
  statusCode = args.statusCode
  breezyHrError = args.error

  rateLimit = call MapRateLimit(headers = args.headers)

  return if (statusCode === 400) {
    title = 'Bad request'
    detail = breezyHrError
    code = 'BadRequest'
    rateLimit = rateLimit
  }

  return if (statusCode === 401) {
    title = 'Unauthenticated'
    detail = breezyHrError
    code = 'Unauthenticated'
    rateLimit = rateLimit
  }

  return if (statusCode === 403) {
    title = 'Unauthorized'
    detail = breezyHrError
    code = 'Unauthorized'
    rateLimit = rateLimit
  }

  return if (statusCode === 404) {
    title = 'Not found'
    detail = breezyHrError
    code = 'NotFound'
    rateLimit = rateLimit
  }

  return if (statusCode === 429) {
    title = 'Rate limit exceeded'
    detail = 'You reached max requests quota.'
    code = 'RateLimitReached'
    rateLimit = rateLimit
  }

  return {
    title = "Unknown error"
    detail = breezyHrError
    code = 'UnknownError'
    rateLimit = rateLimit
  }
```

**Review by martinalbert — APPROVED** [review]
Looks good, just few little things.

**Pull request merged**